src/vse_sync_pp/heatmap.py

```diff
@@ -0,0 +1,64 @@
+### SPDX-License-Identifier: GPL-2.0-or-later
+
+"""Heatmap data parsed from log messages from a single source."""
+
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.patches as patches
+
+
+class Heatmap():
+    """
+    Heatmap visualization for state transitions
```

Comment:
An attempt at trying to generalize the description of the heatmap:
```suggestion
    Heatmap to visualize data values across two axis variables as a grid of colored squares.
```

Reply:
Done